mmi/runner.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 """
 Usage:
-  mmi-runner <engine> <configfile> [-o <outputvar>...] [-g <globalvar>...] [--interval <interval>] [--disable-logger] [--pause] [--mpi <method>] [--track <server>] [--port <port>]
+  mmi-runner <engine> <configfile> [-o <outputvar>...] [-g <globalvar>...] [--interval <interval>] [--disable-logger] [--pause] [--mpi <method>] [--track <server>] [--port <port>] [--bmi-class]
```

Comment:
It just says that engine is a python class. Otherwise <engine> should be made optional, like: (<engine> | --python-bmi-class <class>). Having both engine and class does not sound like a nice idea. Engine is also quite ambigious, i would suggest to switch to (--bmi-library <library> | --bmi-python-class <class>).